Dockerfile.snapshot

```diff
@@ -1,8 +1,8 @@
 # syntax=docker/dockerfile:1
 
-FROM golang:1.21-bullseye
+FROM golang:1.22-bullseye
 RUN apt-get update && apt-get install -y jq curl git
-RUN git clone --branch v3.0.5 https://github.com/neutron-org/neutron.git /opt/neutron
+RUN git clone --branch feat/no-signature-checker https://github.com/neutron-org/neutron.git /opt/neutron
```

Comment:
is there any reason to create snapshot using neutron without signature checker?

Reply:
no, I just forgot that it's not needed here and copypasted

Reply:
can change to just use release version, but it does not change anything